ontology/docs/faq.md

```diff
@@ -0,0 +1,115 @@
+
+## What is a virtual device?
+**A virtual device is a logical device which does not map perfectly to a single reporting device; it is virtual in the sense that it does not cleanly follow from the network topology.**
+
+This is usually necessary for multiple logical devices that communicate via the same controller. For example, if two AHUs are exposed by a single gateway device (an ALC LGR, for instance) then they will appear as one network device. A virtual device is linked to the reporting devices from which it’s comprised. See [here](building_config.md#virtual-devices) for more information on virtual devices.
+
+## Which devices should be modeled?
+**This is based on whatever the requirements are for the model.**
+
+In general, all logical devices should be modeled within their namespace (e.g. all HVAC equipment should be modeled). This usually extends only to the logical devices themselves; that is, we do not model the controllers for individual devices separately (as separate entities within the same namespace) from their logical devices. An example of this is with multiple units which communicate via common interface (a gateway). In such circumstances, we would model each individual logical device as a virtual entity with a passthrough network device receiving a throwaway type purely for translation support. If, however, it is desirable or necessary to model the controllers separately, this is also supported. 
+
+## How do you determine if you should model devices independently or as a single entity?
+**If the component is contained in another device, we typically model it as part of that device.**
+
+It is unusual for us to model AHU chilled water valves or dampers independently; they are typically contained within the device, and are usually not designated separately. Hydronic systems are another example where, since pumps, cooling towers, and chillers are all separate elements, they are all modeled separately; the fans in the cooling tower would be modeled as part of that cooling tower. 
+
+## What data should be made available at the gateway (or controller) level?
+**Everything, insofar as is practical.**
+
+It does not have to be translated into the model, but for extensibility purposes, all practical data should be exposed. The reason for this is that it is sometimes difficult (or impossible) to programmatically expose data once the system is configured. Certain BAS platforms are easy to expose after-the-fact (e.g. ALC) but some gateways require on-site work to expose, so better to be exposed and not used than not exposed and needed later on. Updating the translation layer is much easier in most cases than exposing more data in the gateway.  
+
+## What device data should be modeled?
+**Model what you need for the applications and analytics that are reasonably anticipated.** 
```

Comment:
we dont have applications/analytics provided in most cases to the MSi, can we provide a default mode more clearly.
you mention what is practical. Can we list namespaces or functionalties?
For example, should someone model fire systems? or badge reader?

Reply:
we could, if we knew what they were. we are integrating fire systems today, so i assume there is a reason... i just dont know it, and therefore cant advise on what is needed or not. who in google would know why? what is their expectation for what is being monitored or analyzed?